core/src/main/java/org/apache/sdap/mudrod/main/MudrodEngine.java

```diff
@@ -391,12 +391,12 @@ public static void main(String[] args) {
       me.end();
     } catch (Exception e) {
       HelpFormatter formatter = new HelpFormatter();
-      formatter.printHelp("MudrodEngine: 'dataDir' argument is mandatory. " + "User must also provide an ingest method.", true);
+      formatter.printHelp("MudrodEngine: 'dataDir' argument is mandatory. " + "User must also provide an ingest method.", options, true);
```

Comment:
This code is buggy. We just want to throw the Exception... not continuously print the help arguments.

Reply:
@lewismc In my computer, the old code (
formatter.printHelp("MudrodEngine: 'dataDir' argument is mandatory. " + "User must also provide an ingest method.", true))  comes with an  error “The method printHelp(String, Options) in the type HelpFormatter is not applicable for the arguments (String, boolean)”